taxonomies/ingredients.txt

```diff
@@ -49675,15 +49674,16 @@ es:horchata de chufa
 #
 ###################################################################################################
 <en:nut
-en:tree nuts, treenuts
+en:tree nut, treenuts, tree nuts
 es:nueces de árboles
+allergens:en: en:nut
```

Comment:
we need to use the canonical name in the allergens taxonomy (in plural for nuts)

```suggestion
allergens:en: en:nuts
```